release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -549,6 +549,62 @@ For clusters on user-provisioned xref:../installing/installing_bare_metal/instal
+
+This feature automatically translates in-tree objects to their counterpart CSI representations and should be completely transparent to users. Translated objects are not stored on disk, and user data is not migrated.
+
+While storage class referencing to the in-tree storage plug-in will continue working, it is recommended that you switch the default storage class to the CSI storage class.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-migration.adoc[CSI automatic migration].
+
+[id="ocp-4-11-storage-csi-migration-azure-disk-ga"]
+==== Automatic CSI migration for Microsoft Azure Disk is generally available
+Starting with {product-title} 4.8, automatic migration for in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers became available as a Technology Preview feature. Support for Azure Disk was provided in this feature in {product-title} 4.9, and {product-title} 4.11 now supports automatic migration for Azure Disk as generally available. CSI migration for Azure Disk is now enabled by default and requires no action by an administrator.
+
+This feature automatically translates in-tree objects to their counterpart CSI representations and should be completely transparent to users. Translated objects are not stored on disk, and user data is not migrated.
+
+While storage class referencing to the in-tree storage plug-in will continue working, it is recommended that you switch the default storage class to the CSI storage class.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-migration.adoc[CSI automatic migration].
+
+[id="ocp-4-11-storage-csi-resize-ga"]
+==== Expanding CSI volumes is generally available
+Starting with {product-title} 4.3, expansion of Container Storage Interface (CSI) storage volumes after they have already been created became available as a Technology Preview feature, and is now generally available in {product-title} 4.11.
+
+For more information, see xref:../storage/expanding-persistent-volumes.adoc#expanding-csi-volumes_expanding-persistent-volumes[Expanding CSI volumes].
+
+[id="ocp-4-11-storage-csi-ephemeral_vols"]
+==== Support for CSI generic ephemeral volumes is generally available
+{product-title} 4.11 supports Container Storage Interface (CSI) generic ephemeral volumes as generally available. Generic ephemeral volumes are a type of ephemeral volume that can be provided by all storage drivers that also support persistent volumes and dynamic provisioning.
+
+For more information, see ???
```

Comment:
I think this is the only remaining reference that needs to be updated, but the rest looks good to me.

Reply:
You are correct. After I merge Gen Ephemeral Vols PRs, then I will update the rel notes xref.